numba/core/config.py

```diff
@@ -220,7 +220,7 @@ def optional_str(x):
         # (up to and including IR generation)
         DEBUG_FRONTEND = _readenv("NUMBA_DEBUG_FRONTEND", int, 0)
 
-        # Enable debug prints in nrtdynmod
+        # Enable debug prints in nrtdynmod and use of "safe" API functions
```

Comment:
I was going to suggest updating the documentation for `NUMBA_DEBUG_NRT` but it appears not to be documented at all. Is it worth adding a note on this to the documentation (in the [list of environment variables](https://numba.readthedocs.io/en/stable/reference/envvars.html)) and/or in the [Debugging Leaks](https://numba.readthedocs.io/en/stable/developer/numba-runtime.html#debugging-leaks) section?